taxi/models.py

```diff
@@ -27,6 +28,22 @@ def __str__(self):
     def get_absolute_url(self):
         return reverse("taxi:driver-detail", kwargs={"pk": self.pk})
 
+    def clean(self):
+        if (
+                len(self.license_number) != 8
+                or not self.license_number[:3].isupper()
+                or not self.license_number[:3].isalpha()
+                or not self.license_number[-5:].isdigit()
```

Comment:
The validation logic in the `clean` method for `license_number` is correct, but it could be simplified. The check `not self.license_number[:3].isupper()` is redundant because `isalpha()` already ensures that the characters are letters. Consider removing the `isupper()` check or combining it with `isalpha()` to ensure the first three characters are uppercase letters.